linera-core/src/client/mod.rs

```diff
@@ -807,9 +807,9 @@ impl<Env: Environment> Client<Env> {
         let remote_certificates = remote_node
             .download_certificates(certificate_hashes)
             .await?;
+        let mut certificates = BTreeMap::new();
```

Comment:
The type here is `BTreeMap<ChainId, BTreeMap<BlockHeight,GenericCertificate<ConfirmedBlock>>>`.
That is somewhat complex and the name does not give a hint that it would be that.

What about `certificates_by_height_by_chain` or something similar.

Reply:
Done in https://github.com/linera-io/linera-protocol/pull/4026/commits/a2ee3d9a4bb37472012910e5aafe97dc60f15457.